docs/Agent/user-guide/man_page.md

```diff
@@ -1 +1,472 @@
+
+`--help`  
+Show this message and exit.
+
+---
+
+### pbench-register-tool-set
+
+---
+
+**NAME**  
+`pbench-register-tool-set`, Register the specified toolset
+
+**SYNOPSIS**  
+`pbench-register-tool-set [--toolset=<tool-set>] [--group=<group-name>] [--interval=<interval>] [--no-install] [--remotes=<remote-host>[,<remote-host>]] [--labels=<label>[,<label>]] [<tool-set>]`
+
+`pbench-register-tool-set [--toolset=<tool-set>] [--group=<group-name>] [--interval=<interval>] [--no-install] [--remotes=@<remotes-file>] [<tool-set>]`
+
+**DESCRIPTION**  
+Register all the tools in the specified toolset.
+
+**OPTIONS**  
+ `--toolset`  
+Available tool sets from /opt/pbench-agent/config/pbench-agent.cfg:
+
+- heavy
+- legacy
+- light
+- medium
+
+`--remotes`  
+Single remote host, a list of remote hosts (comma-separated, no spaces) or an "at" sign (`@`) followed by a filename. In this last case, the file should contain a list of hosts and their (optional) labels. Each line of the file should contain a hostname, optionally followed by a label separated by a comma (`,`); empty lines are ignored, and comments are denoted by a leading hash, or pound (`#`), character.
+
+`--labels`  
+Where the list of labels must match the list of remotes.
+
+`--help`  
+Show this message and exit.
+
+---
+
+### pbench-register-tool-trigger
+
+---
+
+**NAME**  
+`pbench-register-tool-trigger`, Registers the tool trigger
+
+**SYNOPSIS**  
+`pbench-register-tool-trigger [OPTIONS]`
+
+**DESCRIPTION**  
+Registers tool with the given group and start and stop tool trigger text
+
+**OPTIONS**  
+ `-C`, `--config` PATH  
+Path to the Pbench Agent configuration file.
+This option is required if not provided by the `_PBENCH_AGENT_CONFIG` environment variable.
+
+`-g`, `--group`, `--groups` STR  
+Registers the trigger in the <`STR`> group. If no group is specified, the `default` group is assumed. 
+
+`--start-trigger` STR  
+[required]
+
+`--stop-trigger` STR  
+[required]
+
+`--help`  
+Show this message and exit.
+
+---
+
+### pbench-results-move
+
+---
+
+**NAME**  
+`pbench-results-move`, Move results tarball to the server to a 1.0 Pbench Server.
+
+**SYNOPSIS**  
+`pbench-results-move [OPTIONS]`
+
+**DESCRIPTION**  
+This command uploads one or more result directories to the configured v1.0 Pbench Server. The specified API Key is used to authenticate the user and to establish ownership of the data on the server. Once the upload is complete, the result directories are, by default, removed from the local system.
+
+
+**OPTIONS**  
+ `-C`, `--config` PATH  
+Path to the Pbench Agent configuration file.
+This option is required if not provided by the `_PBENCH_AGENT_CONFIG` environment variable.
+
+`--controller` STR  
+Override the default controller name
+
+`--token` STR  
+pbench server authentication token [required]
```

Comment:
There's no actual rule that it's an API Key, although using an OIDC token here would be awkward due to the rapid expiration. I thought "authentication token" was fine...

Reply:
I suppose that "authentication token" is OK; but, since the command description references "API Key" (which I think _is_ appropriate), it seemed like the text here should do the same.  (The problem is that, if we're too general, then we're going to need to supply text which explains what an "authentication token" is and where/how to get one...whereas, I'm hoping that if we go with "API Key" the user will be able to find his/her way to the UI (or to its documentation, at least).